### DISCUSSION TITLE
Sometimes can find server; sometimes can't

### DISCUSSION THREAD

**Discussion by SteveJames1968** [apohl79/audiogridder#1279] [General]
Hi all

First, thank you for creating AG

I have AG installed on my W7 computer, and the server installed on my W11 laptop. They are both plugged into the same Lynksis router, and both have different static ip addresses: 192.168.1.100 and 192.168.1.102

My first attempt at setting up was successful; I was able to load the VSL Synchron player in the AG server (W11), and I can send MIDI there, and it sends the audio back to me. Great.

I at times have an issue, however. Sometimes, I am good, but sometimes, AG on my W7 computer can't find the server. Some of those times, I can connect manually with servers > server name > connect. But sometimes there is no server name in the list; it's just empty. I try to use servers > add but that doesn't work. 

The problem occurs when I power down the laptop. If I leave it on, I can power down the W7 master again and again, and AG will ALWAYS connect to the W11 server. But if I power down the laptop, this happens about 75% of the time. Ideas? Thanks!

**Comment by SteveJames1968** [comment]
I cant get this working now. I tried an uninstall/reinstall, and now I can't connect at all. I can add the laptop, and I see the ip address under 'servers', but it will just not connect.  Where can I look?

**Comment by apohl79** [comment]
You can check the plugin logfiles for any clues (https://audiogridder.com/bug-report-diagnostics-locations/). Most of the time those issues are related to the firewall. Try disabling it (temporarily) and see if the connection works.

**Reply by SteveJames1968** [reply]
Checked the log files - no clues there. FW has been disabled. It's working now, but I don't understand why sometimes it does and sometimes it doesn't.
